common/shlibs

```diff
@@ -715,6 +715,7 @@ libnetpgp.so.0 libnetpgp-20100313_1
 libmj.so.0 libmj-20140211_1
 libmca_common_sm.so.40 libopenmpi-3.0.1_1
 libmca_common_ompio.so.41 libopenmpi-3.0.1_1
+libmca_common_monitoring.so.50 libopenmpi-3.1.2_1
```

Comment:
Is this required by some package ? if not please don't add

Reply:
Yes, it is required by the openmpi main package.